### PR TITLE
Feature/trll 39

### PR DESCRIPTION
**Pull request by BtencateSphereon**
Added menu option "User dashboard" that directs to Alfresco share

### CODE REVIEW

File: src/assets/i18n/ja.json [hunk 1]
```diff
@@ -1,6 +1,7 @@
 {
   "APP": {
     "LANGUAGE": "言語",
+    "USER_DASHBOARD": "ユーザーダッシュボード」",
```
Comment (LDMGN) [inline src/assets/i18n/ja.json]: We might have somebody in Sphereon who speaks Spanish and perhaps even Italian, but where did we get the Asian, Scandinavian and Russian translations from?

Comment (BtencateSphereon) [reply src/assets/i18n/ja.json]: @LDMGN google translate

Comment (LDMGN) [reply src/assets/i18n/ja.json]: German and english are fine. Please check french with @ChristopheVico. I think that the dutch version should be "Gebruikerspaneel". The ones that we are not sure about should be either removed or replaced with the english version (depending on how Alfresco handles missing values and defaults).

Comment (BtencateSphereon) [reply src/assets/i18n/ja.json]: @LDMGN removed the uncertain ones and renamed the one for dutch to "Gebruikerspaneel"

**Review by LDMGN — APPROVED** [review]

**Pull request merged**